docs/security/tokens/overview.md

```diff
@@ -1,22 +1,186 @@
+it could have a token for read access to an input dataset, and a token for write access to a results directory.
+
+
+Token Components
+----------------
+
+Bearer tokens are credential strings in the [JSON Web Token (JWT)](https://jwt.io) format.
+A JWT consists of a JSON header, a JSON payload, and a signature that can be verified.
+The payload contains a number of fields, called "claims", that describe the token and what it can access.
+
+There are two JWT-based token standards that can be used with OSG software: [SciTokens](https://scitokens.org)
+and [WLCG Tokens](https://github.com/WLCG-AuthZ-WG/common-jwt-profile/blob/master/profile.md).
+These standards describe the claims that are used in the payload of the JWT.
+
+SciTokens and WLCG Tokens are similar standards and have some common claims:
+
+**Issuer ("iss")**
+
+The issuer identifies the organization that issued the token.
+An issuer looks like an HTTPS URL;
+this URL must be valid and publicly accessible because it is used by services to validate the token.
+Token issuers will be described below.
+
+**Subject ("sub")**
+
+The subject identifies an entity (which could be a human or a robot) that owns the token.
+Unlike the subject of an X.509 certificate, a token subject does not need to be globally unique,
+only unique to the issuer.
+Subjects will be elaborated on below.
+
+**Issued-at ("iat"), not-before ("nbf"), expiration ("exp")**
+
+These claims are Unix timestamps that specify when the token was issued, and its lifespan.
+
+**Audience ("aud")**
+
+The audience is a server (or a JSON list of servers) that the token may be used on;
+it is typically a hostname, host:port, or URI.
+For example a token used for submitting a job to a CE would have
+`<CE FQDN>:<CE PORT>` in the `aud` claim.
+The special values `ANY` (SciTokens) or `https://wlcg.cern.ch/jwt/v1/any` (WLCG Tokens) allow the token to be
+used on any server.
+
+**Scope ("scope")**
+
+The scope limits the actions that can be made using the token.
+The format of the scope claim differs between SciTokens and WLCG Tokens;
+scopes in use by OSG services will be listed below.
+    
+
+### Issuer ###
+
+To generate bearer tokens, a collaboration must adminster at least one "token issuer" to issue tokens to their users.
+In addition to generating and signing tokens, token issuers provide a public endpoint that can be used to validate an
+issued token,
+e.g. an OSG Compute Entrypoint (CE) will contact the token issuer to authorize a bearer token used for pilot job
+submission.
+
+The issuer is listed in the `iss` claim; this should be an HTTPS URL of a web server.
+This server must have the public key that can be used to validate the token in a well-known location,
+as described by the [OpenID Connect Discovery standard](https://openid.net/specs/openid-connect-discovery-1_0.html).
+If the issuer is down, or the the public key cannot be downloaded, the token cannot be verified
+and will be rejected.
```

Comment:
One thing to note is that clients of a token issuer will cache the public key for a few hours (and technically that cache is 4 days right now to help ease the transition to tokens while collabs spin up their token issuers)

Reply:
Added